src/Slider.tsx

```diff
@@ -95,49 +95,43 @@ export const Slider = forwardRef<SliderTypes.API, PropsWithChildren<Settings>>((
     const mouseUpHandler = () => setIsDragging(false);
 
     const mouseDownHandler = (event: ReactMouseEvent<HTMLDivElement>) => {
-        setMousePosition({
+        mousePosition.current = {
             ...mousePosition,
             clientX: event.clientX,
             clientY: event.clientY,
             scrollX: wrapper.current?.scrollLeft ?? 0,
             scrollY: wrapper.current?.scrollTop ?? 0,
-        });
+        };
 
         setIsDragging(true);
     };
 
     const mouseMoveHandler = (event: ReactMouseEvent<HTMLDivElement>) => {
         const currentWrapper = wrapper.current;
+        const currentMousePos = mousePosition.current;
```

Comment:
```suggestion
        const currentMousePosition = mousePosition.current;
```

Beter write out the whole variable name, easier to understand.